docs/en/observability/synthetics-analyze.asciidoc

```diff
@@ -0,0 +1,358 @@
+included the in <<synthetics-analyze-one-run>> section below.
+
+[role="screenshot"]
+image::images/synthetics-analyze-individual-monitor-history.png[The History tab on the individual monitor page for all monitor types in the Synthetics UI]
+
+[discrete]
+[[synthetics-analyze-individual-monitors-errors]]
+== Errors
+
+The *Errors* tab has information on every time a run has failed.
+It includes a high-level overview of all alerts and a complete list of all run failures.
+Use the image:images/icons/calendar.svg[Calendar icon] and search bar to filter for runs
+that occurred in a specific time period.
+
+// What you might do with this info
+// ...
+
+For browser monitors, you can click on any run in the *Error* list
+to open an *Error details* page that includes most of the same information
+that is included the in <<synthetics-analyze-one-run>> section below.
+
+[role="screenshot"]
+image::images/synthetics-analyze-individual-monitor-errors.png[The Errors tab on the individual monitor page for all monitor types in the Synthetics UI]
+
+[discrete]
+[[synthetics-analyze-journeys]]
+= Browser monitors
+
+For browser monitors, you can look at results at various levels of granularity:
+
+* See an overview of journey runs over time.
+* Drill down into the details of a single run.
+* Drill down further into the details of a single _step_ within a journey.
+
+[discrete]
+== Journey runs over time
+
+The journey page includes:
+
+* An overview of the *last test run* including high-level information for each step.
+* *Alerts* to date including both active and recovered alerts.
+* *Duration by step* over the last 24 hours.
+* A list of the *last 10 test runs* that link to the <<synthetics-analyze-one-run, details for each run>>.
+
+[role="screenshot"]
+image::images/synthetics-analyze-journeys-over-time.png[Individual journey page for browser monitors in the Synthetics UI]
+
+From here, you can either drill down into:
+
+* The latest run of the full journey by clicking *image:images/icons/inspect.svg[Inspect icon] View test run*
+  or a past run in the list of *Last 10 test runs*.
+  This will take you to the view described below in <<synthetics-analyze-one-run>>.
+* An individual step in this run by clicking *image:images/icons/apmTrace.svg[Performance breakdown icon]*
+  next to one of the steps. This will take you to the view described below in <<synthetics-analyze-one-step>>.
+
+[discrete]
+[[synthetics-analyze-one-run]]
+== Details for one run
+
+The page detailing one run for a journey includes more information on each step in the current run
+and opportunities to compare each step to the same step in previous runs.
+
+// What info it includes
+At the top of the page, see the _Code executed_ and any _Console_ output for each step.
+If the step failed, this will also include a _Stacktrace_ tab that you can use to
+diagnose errors.
+
+Click through each step using *image:images/icons/arrowLeft.svg[Previous icon] Previous* and
```

Comment:
nit maybe instead of "Click through each...", use "Navigate through each..." (to distinguish it from a click through)